types/protobuf.go

```diff
@@ -124,10 +124,10 @@ func (tm2pb) Evidence(ev Evidence, valSet *ValidatorSet) abci.Evidence {
 	}
 
 	// set type
-	var evType string
+	var evType abci.EvidenceType
 	switch ev.(type) {
 	case *DuplicateVoteEvidence:
-		evType = ABCIEvidenceTypeDuplicateVote
```

Comment:
should remove lines 17-19 in this file

Reply:
lol you couldn't write the message there so you did it here. Ahh the faults in github. Thanks for the heads up though